src/main/scala/com/amazon/deequ/analyzers/jdbc/JdbcCorrelation.scala

```diff
@@ -103,32 +104,29 @@ case class JdbcCorrelation(firstColumn: String,
 
     val result = statement.executeQuery()
 
-    try {
```

Comment:
Strange, why is try/catch missing here?

Reply:
because it's not included in any analyzer